src/screens/payment-methods/PaymentMethods.tsx

```diff
@@ -72,7 +78,7 @@ export default class PaymentMethods extends BaseScreen<Props, State> {
       };
       const paymentMethods = await this.centralServerProvider.getPaymentMethods(params, { skip, limit });
       // Get total number of records
-      if (paymentMethods.count === -1) {
+      if (paymentMethods?.count === -1) {
```

Comment:
Sometimes the items length count is -1, idk why

Reply:
Mais LOL !!!!
@AlixH  Find out why it's negative (usually it's because the number of items > 500) but this should not occur in that case AND deliver the proper code.
YOU MUST ALWAYS KNOW WHAT YOU'RE DOING WHEN YOU CODE.
Violation of Golden rule #7: understand the purpose of every line of code you use

Reply:
All clear now

Reply:
> All clear now

Perfect :-)